### PR TITLE
Fix Boolean constraint

### PR DESCRIPTION
**Pull request by mitschabaude**
`berkeley` version of the fix described here: https://github.com/MinaProtocol/mina/pull/15296

### CODE REVIEW

**Review by mrmr1993 — APPROVED** [review]

**Comment by mrmr1993** [comment]
!ci-build-me

**Comment by mrmr1993** [comment]
!approved-for-mainnet

**Comment by SanabriaRusso** [comment]
!ci-toolchain-me

**Comment by mrmr1993** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by mitschabaude** [comment]
!ci-build-me

**Comment by mitschabaude** [comment]
!ci-build-me

**Comment by mrmr1993** [comment]
!ci-build-me

**Comment by mrmr1993** [comment]
!approved-for-mainnet

**Review by emberian — APPROVED** [review]

**Comment by mrmr1993** [comment]
!ci-build-me

**Comment by emberian** [comment]
!ci-build-me

**Comment by deepthiskumar** [comment]
!ci-build-me

**Comment by emberian** [comment]
!ci-build-me

**Comment by deepthiskumar** [comment]
!approved-for-mainnet

**Comment by deepthiskumar** [comment]
!ci-nightly-me

**Comment by deepthiskumar** [comment]
https://buildkite.com/o-1-labs-2/mina-end-to-end-nightlies/builds/1666

File: buildkite/src/Jobs/Test/DaemonUnitTest.dhall [hunk 1]
```diff
@@ -36,6 +36,7 @@ Pipeline.build
         S.strictlyStart (S.contains "src"),
         S.strictly (S.contains "Makefile"),
         S.exactly "buildkite/src/Jobs/Test/DaemonUnitTest" "dhall",
+        S.exactly "buildkite/src/Constants/ContainerImages" "dhall",
```
Comment (emberian) [inline buildkite/src/Jobs/Test/DaemonUnitTest.dhall:39]: zomg

Comment (SanabriaRusso) [reply buildkite/src/Jobs/Test/DaemonUnitTest.dhall:39]: 🤷‍♂️

**Review by SanabriaRusso — APPROVED** [review]

**Pull request merged**